Helm/opensearch-dashboards/values.yaml

```diff
@@ -0,0 +1,136 @@
+# Default values for opensearch-dashboards.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+opensearchHosts: "https://opensearch-cluster-master:9200"
+replicaCount: 1
+
+image: "opensearchproject/opensearch-dashboards"
+imageTag: "latest"
+imagePullPolicy: "IfNotPresent"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+
+rbac:
+  create: true
+
+
+podAnnotations: {}
+
+extraEnvs:
+  - name: "NODE_OPTIONS"
+    value: "--max-old-space-size=1800"
+
+podSecurityContext: {}
+
+securityContext:
+  capabilities:
+    drop:
+      - ALL
+  # readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  runAsUser: 1000
+
+
+config:
+  dashboards.yml:
+    server:
+        name: dashboards
+        host: 0.0.0.0
+
+    ## Dashboards TLS Config
+        ssl:
+          enabled: true
+          key: /usr/share/dashboards/certs/dashboards-key.pem
+          certificate: /usr/share/dashboards/certs/dashboards-crt.pem
```

Comment:
My assumption would be that Dashboards would be hosted in the same namespace of as the other OpenSearch components and all communication would presumably be via the internal k8s namespacing. Ex; `opensearch.opensearch.svc.cluster.local`

With this, should we disable the SSL ref? By default in the Opensearch chart we disable SSL for comms (except transport) as everything is blocked off by the Ingress.

Reply:
As per @peterzhuamazon we wanted to have SSL enabled by default so I had enabled SSL by default even for OpenSearch.

Reply:
@peterzhuamazon Any reason for this? Transport SSL makes sense between the nodes. 

But 99% of people consuming it over HTTP will be using a secure proxy / service mesh to bring in traffic over TLS/SSL.

Reply:
So by default OpenSearch have HTTPS enable by default but on Dashboards we have HTTP by default.
Same back in the days of ODFE where ES have HTTPS and KIBANA have HTTP.

Reply:
@peterzhuamazon so you want SSL to be disabled by default for Dashboards I suppose?

Reply:
Yes.